### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by Mariangela**
Updated readme after tested the steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,176 +49,296 @@ Using WSO2 adapter, users can do the following.
...
-- Access WSO2 API Manager Publisher and Store for analytics
+#### Step 1 - Install [Istio 1.1 or above](https://istio.io/docs/setup/kubernetes/install/)
+**NOTE:** 
+- Make sure the Kubernetes (K8s) cluster has at least 8GB of memory.
+-   In the default profile of Istio installation, the policy check is disabled by default. However, in a production environment it is mandatory to use the Mixer Adapter; therefore, in such a scenario you need to explicitly enable the policy check. For more information, see [Enable Policy Enforcement](https://istio.io/docs/tasks/policy-enforcement/enabling-policy/)
+
+#### Step 2 - Install WSO2 API Manager Analytics
+1. Install [Istio-apim release: wso2am-istio-1.0.zip](https://github.com/wso2/istio-apim/releases/tag/1.0).
+The wso2am-istio-1.0.zip contains installation artifacts that you need to deploy in Istio, WSO2 API Manager, and WSO2 API Manager Analytics as explained in the subsequent steps.
+2. Extract wso2am-istio-1.0.zip and navigate to the <APIM-ISTIO-HOME>/ directory.
```
Comment (pubudu538) [inline README.md]: APIM-ISTIO-HOME is not available in the readme due to the formatting issues. Only available the following lines.  Extract wso2am-istio-1.0.zip and navigate to the / directory.

File: README.md [hunk 2]
```diff
@@ -49,176 +49,296 @@ Using WSO2 adapter, users can do the following.
...
+   `
+   cd <APIM-ISTIO-HOME>/
+   `
+   **Note:** You need to run all `kubectl` commands from within the <APIM-ISTIO-HOME>/ directory.
```
Comment (pubudu538) [inline README.md]: Note should start in a new line.

File: README.md [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
 
 # Background
 
-When users move towards microservice architecture from monolithic app architecture, it can result in a considerable number of fine-grained microservices. So, it was a challenge to manage all these microservices. As a solution, Istio was able to provide a platform to connect, manage and secure all these microservices while reducing the complexity of deployments. In addition, Istio includes APIs that let it integrate into any logging platform, or telemetry or policy system.
+When users move towards a microservice architecture from monolithic app architecture, it can result in a considerable number of fine-grained microservices. Therefore, it was a challenge to manage all these microservices. As a solution, Istio was able to provide a platform to connect, manage, and secure all these microservices, while reducing the complexity of deployments. In addition, Istio includes APIs that let it integrate in to any logging platform, telemetry, or policy system.
```
Comment (fazlan-nazeem) [inline README.md:7]: I think 'into' is the correct word to use here

**Review by pubudu538 — COMMENTED** [review]
Thank you Mari for contributing to the project.

**Review by pubudu538 — APPROVED** [review]

**Pull request merged**